### PR TITLE
Cleanup the CLI commands

### PR DESCRIPTION
**Pull request by josephlewis42**
Refactoring the CLI commands into init() functions so they doesn't litter the namespace with single-use variables.

There are two intentional functionality changes in here:

* You can now dump the operations table too in the database dumping sub-command.
* The API client now uses `log.Fatalf` rather than returning an error, to Cobra returning an error means a configuration issue so it prints out the help of the command too which caused somewhat confusing errors.

### CODE REVIEW

File: cmd/client.go [hunk 1]
```diff
@@ -159,10 +149,18 @@ func newClientCommand(use, short string, run func(*client.Client) *client.Broker
 		RunE: func(cmd *cobra.Command, args []string) error {
```
Comment (craigatgoogle) [inline cmd/client.go]: This should be changed to Run:

File: cmd/client.go [hunk 2]
```diff
@@ -174,13 +172,3 @@ func createClient() (*client.Client, error) {
 
 	return client.New(user, pass, "localhost", port)
 }
-
-func printJsonResults(results interface{}) error {
```
Comment (craigatgoogle) [inline cmd/client.go]: This JSON formatting functionality seems to be repeated throughout the PR. Any reason in particular to not have it live in this helper function?

Comment (josephlewis42) [reply cmd/client.go]: Nice catch, I pulled it out into the util package.

File: utils/utils.go [hunk 3]
```diff
@@ -61,3 +62,15 @@ func MergeStringMaps(map1 map[string]string, map2 map[string]string) map[string]
 	}
 	return combined
 }
+
+// PrettyPrintOrExit writes a JSON serialized version of the content to stdout.
+// If a failure occurs during marshaling, the error is logged along with a
+// formatted version of the object and the program exits with a failure status.
+func PrettyPrintOrExit(content interface{}) {
```
Comment (craigatgoogle) [inline utils/utils.go]: Perhaps make whether or not to exit an argument, that way this method will still work for the other use-cases throughout this PR which need this behavior but with a returned error.

Comment (craigatgoogle) [reply utils/utils.go]: Or else, have a separate method which returns an error, and have both use a private helper.

Comment (craigatgoogle) [reply utils/utils.go]: The second way sounds cleaner the more I think about it.

Comment (josephlewis42) [reply utils/utils.go]: I like that, it'll also go cleanly with the way the rest of this software is structured.

**Pull request merged**